update.sh

```diff
@@ -3,7 +3,7 @@ set -eo pipefail
 
 declare -A php_version=(
 	[default]='7.2'
-	[12.0]='7.1'
+	[15.0]='7.3'
 )
```

Comment:
I think you should make 7.3 the default and set 13 and 14 to 7.2.
Because if nextcloud 16 is ready we would want it to use php 7.3 and not 7.2